### PR TITLE
[MDCT-2638] logout-redirect

### PR DESCRIPTION
**Pull request by BearHanded**
### Description

Create an endpoint to handle redirect on successful logout, and all the config that is required.

The endpoint exists outside the normal auth gates, so returning to it doesn't trap you in an endless loop in prod.

### Related ticket(s)

MDCT-2638

---
### Important updates

---
### Author checklist

- [ ] I have performed a self-review of my code
- [ ] I have added [thorough](https://bit.ly/3zPrxuZ) tests, if necessary
- [ ] I have updated relevant documentation, if necessary
---

_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_

### CODE REVIEW

File: services/ui-auth/serverless.yml [hunk 1]
```diff
@@ -129,7 +129,9 @@ resources:
         DefaultRedirectURI: ${self:custom.application_endpoint_url}
         LogoutURLs:
           - ${self:custom.application_endpoint_url}
```
Comment (gmrabian) [inline services/ui-auth/serverless.yml:131]: do we wanna keep the app endpoint and `localhost:3000/`?

**Review by gmrabian — APPROVED** [review]
lgtm

File: services/ui-auth/serverless.yml (hunk 1 above)
Comment (BearHanded) [reply services/ui-auth/serverless.yml:131]: In my mind I was keeping it so we could easily change it back with minor confusion if we wanted to, but if there's a strong opinion to change it I will

**Review by keeysnc — APPROVED** [review]
looks good!

**Pull request merged**